thoth/storages/graph/models.py

```diff
@@ -52,6 +52,21 @@ class PackageExtractRun(VertexBase):
     duration = model_property(type=int)
 
 
+@attr.s(slots=True)
+class PackageAnalyzerRun(VertexBase):
+    """A class representing a single package-analyzer (package analysis) run."""
+
+    package_analysis_document_id = model_property(type=str, index="exact")
+    package_analysis_datetime = model_property(type=datetime, index="hour")
+    package_analysis_version = model_property(type=str, index="exact")
+    package_analysis_name = model_property(type=str, index="exact")
+    environment_type = model_property(type=str, index="exact")
```

Comment:
@fridex Do we need environment_type? What else we would need to store for package analysis documents?

Reply:
We will need to first finish the initial design of the package-extract output. Then, take a look at [schema](https://raw.githubusercontent.com/thoth-station/storages/master/docs/schema.png) and try to come up with some changes. We can discuss it in a meeting.

Reply:
Got it!

Reply:
:+1: